jobs/kibana-auth-plugin/templates/config/config.sh.erb

```diff
@@ -5,6 +5,7 @@ export KIBANA_OAUTH2_CLIENT_ID="<%= p("kibana-auth.cloudfoundry.client_id") %>"
 export KIBANA_OAUTH2_CLIENT_SECRET="<%= p("kibana-auth.cloudfoundry.client_secret") %>"
 export CF_SYSTEM_ORG="<%= p("kibana-auth.cloudfoundry.system_org") %>"
 export SESSION_EXPIRATION_MS="<%= p("kibana-auth.session_expiration_ms") %>"
+<% if_p("kibana-auth.session_key") do |key| %>export SESSION_KEY="<%= key %>"<% end %>
```

Comment:
WDYT about `raise ArgumentError` here if it isn't >= 32 bits?

This is the approach UAA takes when input properties can be validated during the template rendering phase: https://github.com/cloudfoundry/uaa-release/blob/develop/jobs/uaa/templates/uaa.yml.erb#L605